src/components/common/CPicker/styles.js

```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from 'react-native';
+import { BLACK } from '../../../constants/styleConstants';
```

Comment:
Use the path shortcuts.
```suggestion
import { BLACK } from 'constants/styleConstants';
```